heron/storm/src/java/shade.conf

```diff
@@ -0,0 +1,4 @@
+rule com.google.** org.apache.storm.shaded.@0
+rule com.twitter.** org.apache.storm.shaded.@0
```

Comment:
@nlu90 Timur is right. I am thinking `com.twitter.xxx` is just "implementation" in this context.